### DISCUSSION TITLE
[UserStoriesToRefine] #7 As <user> I want a website where users can easily search or filter videos by topics and language availability

### DISCUSSION THREAD

**Discussion by emecas** [emecas/civic-tech-website-consultancy-pilot-translation#30] [General]
### As <user> I want a website where users can easily search or filter videos by topics and language availability

**Comment by nhrafiq** [comment]
## Level 2: As a cacao producer, I want to be able to filter the video library so that I can find videos relevant to me that are available in my language of interest. 
Acceptance criteria:
* It should be possible to filter the library on a number of different criteria, such as video tags (which may include video topic and language, among others), date posted, community ratings (how much it has been liked/disliked), etc. 
* It should be possible to sort the list of videos by criteria such as highest rated, language preferences (such as Spanish videos before English videos), most recently posted, etc.
